app/workers/shared_code/db.py

```diff
@@ -19,7 +21,24 @@
 )
 
 
-def update_scan_report_status(id: str, status: Status) -> None:
+class UploadStatusConstant(Enum):
+    IN_PROGRESS = "Upload in Progress"
+    COMPLETE = "Upload Complete"
+    FAILED = "Upload Failed"
+
+
+class MappingStatusConstant(Enum):
+    PENDING = "Pending Mapping"
+    COMPLETE = "Mapping Complete"
+    BLOCKED = "Blocked"
+    MAPPING_25PERCENT = "Mapping 25%"
+    MAPPING_50PERCENT = "Mapping 50%"
+    MAPPING_75PERCENT = "Mapping 75%"
+
+
+def update_scan_report_status(
```

Comment:
So when we upload a new SR, without setting the mapping_status along with upkoad_status, it will be null, and cause an error in the front-end app. To avoid this then, I will set the null check in Mapping Status component and set it to "PENDING" if it is null. Thanks ☺️